logger/logger.go

```diff
@@ -69,8 +89,18 @@ func SetLogLevel(name string) {
 	logrus.SetLevel(level)
 }
 
+func (l *Logger) shouldSuppress(message string) bool {
+	if len(l.suppressionFilter) == 0 {
```

Comment:
nit: usually go does empty string checks against `""` instead of `len`, but that's fine too.